js/components/graph/Node.js

```diff
@@ -262,7 +265,18 @@ export default class Node extends React.Component {
     return (
       <g {...gAttrs} id={this.props.JSON.id_} className={newClassName} data-testid={this.getDataTestId()}>
         {ellipse}
-        <rect {...rectAttrs} style={rectStyle} />
+        <filter id="dropshadow" height="130%">
```

Comment:
This is cool! You should define the `filter` once at the root of the svg (in the `Graph` render); you should still be able to refer to it by id for each node, but this way it won't be duplicated once per node.

Reply:
@kenneth-miura great thanks, I think you also need to update the test snapshots.

Reply:
Ah sorry, didn't notice. tests are updated.